File: api/clients/src/main/java/com/foodagram/clients/users/profile/UsersProfileDto.java
package com.foodagram.clients.users.profile;

import lombok.*;

import java.util.Date;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UsersProfileDto {
    private UUID id;
    private UUID usersId;
    private String firstName;
    private String lastName;
    private UUID profilePictureID;
    private String bio;
    private String city;
    private String country;
    private String website;
    private String jobTitle;
    private Date birthDate;
    private String instagramProfile;
    private String facebookProfile;
    private long followingCount;
    private long followersCount;
    private byte[] profilePicture;
}

File: fooadgram-app/models/user/UsersProfileDto.ts
export interface UsersProfileDto {
    id: string;
    usersId: number;
    firstName: string;
    lastName: string;
    profilePictureID: string;
    bio: string;
    city: string;
    country: string;
    website: string;
    jobTitle: string;
    birthDate: Date;
    createdAt: Date;
    instagramProfile: string;
    facebookProfile: string;
    followingCount: number;
    followersCount: number;
    profilePicture: Blob;
}
